GUI/gui.py

```diff
@@ -0,0 +1,197 @@
+#!/usr/bin/env python3
+
+import tkinter as tk
+from tkinter import ttk, messagebox
+from ttkthemes import ThemedStyle
+import subprocess
+import os
+import re
+
+
+# Verifica se lo script è stato eseguito con i permessi di root
+if os.geteuid() != 0:
+    messagebox.showinfo("Popup", "You must run the app with sudo permissions.")
+    exit()
+
+MAN = """--==[ torctl.sh by blackarch.org ]==--
+
+Usage: torctl.sh COMMAND
+
+A script to redirect all traffic through tor network
+
+Commands:
+
+start      - start tor and redirect all traffic through tor
+
+stop       - stop tor and redirect all traffic through clearnet
+
+status     - get tor service status
+
+restart    - restart tor and traffic rules
+
+autowipe   - enable memory wipe at shutdown
+
+autostart  - start torctl at startup
+
+ip         - get remote ip address
+
+chngid     - change tor identity
+
+chngmac    - change mac addresses of all interfaces
+
+rvmac      - revert mac addresses of all interfaces
+
+version    - print version of torctl and exit"""
+
+ABOUT = """Official Repository: blackarch.org
+
+GUI Developer: Aleff (github.com/aleff-github)
+
+Licence: GPLv3"""
+
+CONTACT = """Github: https://github.com/aleff-github
```

Comment:
Please don't include links to your personal socials in the code and claim ownership of torctl

Reply:
I had no intention of claiming ownership of torctl, I simply wanted to include a contact.

Reply:
Removed _Aleff thinks_, released only on About section on GUI Developer